content/ri/release-notes/v1.13.0.md

```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
 weight: 7
 ---
 
+## 1.13.2 (May 2023)
+
+This is the maintenance release of RedisInsight 1.13 (v1.13.2)!
+
+### Fixes:
+- Core:
+  - Added support for exporting connections to be used in RedisInsight V2.
```

Comment:
```suggestion
  - Added support for exporting connections to RedisInsight v2.
```

Reply:
Added support for exporting database connections to easily migrate them to RedisInsight v2 by bulk exporting to a file.